storage_service/static/js/project.js

```diff
@@ -1,7 +1,54 @@
-$(document).ready(function () {
+var getAjaxDataTableOptions = function (options, ajaxSource, headers, filter) {
+  var result = {};
+  for (var k in options) {
+    result[k] = options[k];
+  }
```

Comment:
Bit of a nit but I generally try to avoid single-letter variables. Maybe `key` would work here?